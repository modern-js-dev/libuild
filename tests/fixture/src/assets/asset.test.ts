import { getLibuilderTest } from '@modern-js/libuild-test-toolkit';

describe('fixture:assets', () => {
  it('wav file should bundled', async () => {
    const bundler = await getLibuilderTest({
      root: __dirname,
      asset: {
        limit: 14 * 1024,
      },
    });
    await bundler.build();
    bundler.expectJSOutputMatchSnapshot();
  });
});
