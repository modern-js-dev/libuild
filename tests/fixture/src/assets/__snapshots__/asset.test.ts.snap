// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixture:assets bundle default, copy asset 1`] = `
"// index.ts
import svg from \\"./assets/logo.PGX3QVVN.svg\\";
console.log(svg);
"
`;
